scaleout/stackn/values.yaml

```diff
@@ -274,3 +274,8 @@ minio:
   enabled: true
   host: ""
   secretName: minio-fedn
+
+harbor:
```

Comment:
I would add thhis under the key studio.harbor since otherwise it looks like harbor will be deployed.